### ISSUE TITLE
Missing completion keyword `in` inside let expression

### ISSUE THREAD

**Issue by gayalkuruppu** [ballerina-platform/ballerina-lang#34746]
**Description:**

In the following code snippet, `in` keyword is not suggested. From LS Side (LetExpressionNodeContext.java), the `in` keyword is added to the completionItems.
```
public function foo(int a, float b) {
    let float x1 = b + b, int x2 = a i<cur>
}
```

**Affected Versions:**
SL 2201.0.0 RC1

**Comment by gayalkuruppu** [comment]
blocked by #34901

**Comment by gayalkuruppu** [comment]
Fixed in #35066
